css/elementor-icons.min.css

```diff
@@ -1,2 +1,2 @@
\ No newline at end of file
+/*! elementor-icons - v5.17.0 - 18-12-2022 */
```

Comment:
You run grunt after the new fonts are merged, together with updating the version in the package.json, not before - so the versions will match the dates